assets/js/pages/ClusterDetails/SBDDetails.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { sbdDevicesFactory } from '@lib/test-utils/factories';
+
+import { capitalize } from 'lodash';
+
+import SBDDetails from './SBDDetails';
+
+describe('SBDDetails', () => {
+  it('should render empty state message when sbd devices are empty', () => {
+    const expectedEmptyStateMsg = 'No additional fencing details to display.';
+    const sbdDevices = [];
+
+    render(<SBDDetails sbdDevices={sbdDevices} />);
+    expect(screen.getByText(expectedEmptyStateMsg)).toBeInTheDocument();
+  });
+
+  it.each([['healthy'], ['unhealthy']])(
+    'should render %s sbd device',
+    (healthStatus) => {
+      const expectedSBD = sbdDevicesFactory.buildList(1, {
+        status: healthStatus,
+      });
+      const [{ device: expectedDeviceName, status: expectedHealthStatus }] =
+        expectedSBD;
+      const capitalizedHealthStatus = capitalize(expectedHealthStatus);
+
+      render(<SBDDetails sbdDevices={expectedSBD} />);
+
+      expect(screen.getByText(expectedDeviceName)).toHaveTextContent(
+        expectedDeviceName
+      );
+      expect(screen.getByText(capitalizedHealthStatus)).toHaveTextContent(
```

Comment:
Change this to `expect(screen.getByText(capitalize(expectedHealthStatus))).toBeTruthy()`.
The same concept above